op5build/ninja.logrotate

```diff
@@ -0,0 +1,22 @@
+/var/log/op5/ninja.log {
+    su monitor apache
+    weekly
+    rotate 10
+    compress
+    dateext
+    dateformat -%Y%m%d
+    missingok
+    notifempty
+    create 0664 monitor apache
+}
+/var/log/op5/ninja/* {
```

Comment:
What files here need rotation? I think we only put crash reports in this directory and they definitely shouldn't be rotated.

Reply:
Hi @jacobbaungard as per the task descrption "automatically generated files in /var/log/op5/ninja are not rotated and can, therefore, take up quite a lot of space".
Should these be omitted?

Reply:
I can't discuss with @johanthoren at the moment, as he is on vacation. For now, lets stick to ninja.log - I am pretty sure it doesn't make sense to rotate the files in /var/log/op5/ninja/*

Reply:
updated.Thnx